frontend/packages/console-app/src/components/dashboards-page/status.ts

```diff
@@ -111,3 +115,18 @@ export const getClusterOperatorHealthStatus: GetOperatorsWithStatuses<ClusterOpe
     getOperatorsStatus<ClusterOperator>([co], getClusterOperatorStatusPriority),
   );
 };
+
+export const detectControlPlane: ActionFeatureFlagDetector = (dispatch) => {
+  const promURL = getPrometheusURL({
+    endpoint: PrometheusEndpoint.QUERY,
+    query: 'up{job="apiserver"}',
+  });
+
+  return coFetchJSON(promURL)
```

Comment:
We should check if `PROMETHEUS_BASE_PATH` is set before trying to run the query.